monitoring/count-items-cronjob/alerts.yaml

```diff
@@ -0,0 +1,66 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: count_items_cronjob
+    type: constant
+    value: artesca-data-ops-count-items
```

Comment:
nit: I wonder if we should not take the `artesca-data` prefix from an env as this is linked to the zenko CR, in practice if the CR is named differently the operator should probably rename this alert. This can also be done with the current implementation and the operator setting this var with `{crName}-{jobName}` directly

Reply:
hey JB, do you know how can we get the Zenko CR name from Prometheuse?

Reply:
I don't think taking the name of the CR from prometheus would be the right way even through it could be possible. I would more likely say that the operator should change these alert definitions based on the CR it is reconciling.

Reply:
Since you'll have to deploy the S3Utils alerts anyway in ZKOP you'll be able to pass all the inputs from there. [This](https://github.com/scality/zenko-operator/blob/c392b853d0dd16379062faa316c94dce4387cdfc/pkg/controller/zenko/reconcile_backbeat_alerting.go#L35-L67) is an example of the inputs we pass to the backbeat alerts.

The CR name and namespace is passed by default to the alerts in ZKOP [here](https://github.com/scality/zenko-operator/blob/c392b853d0dd16379062faa316c94dce4387cdfc/pkg/controller/zenko/alert_generator.go#L76-L86), you'll just have to rename the inputs to conform to what we have in ZKOP.

You can also pass `count_items_cronjob` as you already have the value [here](https://github.com/scality/zenko-operator/blob/c392b853d0dd16379062faa316c94dce4387cdfc/pkg/controller/zenko/object_builder.go#L65), and `count_items_job_duration_threshold` should probably be configurable at Zenko CR level, but this is to be done in the ZKOP PR.

Reply:
Thinking about it, you probably don't even need to construct the `owner_name` by hand in the alerts by concatenating `zenkoCRName` at the beginning, there is already a function in ZKOP for that and that we use to get the job names.
https://github.com/scality/zenko-operator/blob/c392b853d0dd16379062faa316c94dce4387cdfc/pkg/controller/zenko/object_builder.go#L361-L363